packages/sw-precaching/src/lib/controllers/revisioned-cache-manager.js

```diff
@@ -53,6 +54,21 @@ class RevisionedCacheManager extends BaseCacheManager {
   addToCacheList({revisionedFiles} = {}) {
     assert.isInstance({revisionedFiles}, Array);
     super._addEntries(revisionedFiles);
+
+    const urlsWithoutRevisionFields = revisionedFiles
+      .filter((entry) => typeof entry === 'string');
+    if (urlsWithoutRevisionFields.length > 0) {
+      logHelper.debug({
+        that: this,
+        message: `Some precache entries don't include a separate revision
```

Comment:
This message is awfully verbose for the point it's trying to make.

"Please ensure the following cache entries are revisioned correctly

  /index.1234.css
  /example/unsafe.html

  If these aren't revisioned (a.k.a. cache busted, your users won't receive updates."